corehq/apps/change_feed/producer.py

```diff
@@ -27,17 +27,28 @@ def producer(self):
         )
         return self._producer
 
-    def send_change(self, topic, change_meta):
+    def send_change(self, topic, change_meta, flush=False):
         message = change_meta.to_json()
         try:
             self.producer.send(topic, bytes(json.dumps(message)))
-            self.producer.flush()
+            if flush:
+                self.producer.flush()
```

Comment:
Can this use `flush_changes` itself? the only difference is `sending` and `flushing` in the message, not sure how useful that is.